.azure-pipelines/templates/automation_test.yml

```diff
@@ -1,6 +1,16 @@
+# Azure DevOps task for automation test using azdev and pytest
+
 parameters:
-  pythonVersion: ''
-  profile: ''
+- name: pythonVersion
+  type: string
+  default: ''
```

Comment:
What does '' mean? How about set default value 3.8?

Reply:
`''` means empty string. I copied it as-is. It also appears here: